discovery/aws/disabled.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build noaws
```

Comment:
Can we make these build tags a little more verbose, like `nodiscovery_aws`?

Reply:
The goal is mainly to not use the aws dependency and reduce binary size. That is why I synced the name with te sigv4 PR.

https://github.com/prometheus/common/pull/366/files